### DISCUSSION TITLE
Welcome to SLAM Discussions!

### DISCUSSION THREAD

**Discussion by halfmanbear** [halfmanbear/SLAM#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheTurbs** [comment]
Does this software have the ability to work with a repository, say one hosted by a group?

**Reply by halfmanbear** [reply]
> the ability to work with a repository, say one hosted by a gro

I've added a `.gitnore` file so you'll be able to clone the repo with;

https://github.com/halfmanbear/SLAM.git

any future updates won't reset the `config.txt` or remove the contents of the `Games` directory.
